### ISSUE TITLE
Language switch not working correctly

### ISSUE THREAD

**Issue by phochste** [LibreCat/LibreCat#305]
- Logout
- Switch on the homepage to a language
- Login
- After login the language is switched back to the original language

As @verheyenkoen  says: maybe better to split the cookie in two : a language and login session cookie

**Comment by nicolasfranck** [comment]
Have a look at https://github.com/LibreCat/LibreCat/tree/set_lang

**Comment by verheyenkoen** [comment]
Added test for this in branch [issues/305](https://github.com/LibreCat/LibreCat-Frontend-Tests/tree/issues/305). Can be merged in master when issue is fixed.

To run test:
```
$ $(npm bin)/cypress run --spec cypress/integration/issues/issue-305.spec.js
```

**Comment by vpeil** [comment]
with the additional commit 5d7f029 on branch `set_lang` the frontend test passes.

@nicolasfranck I also like the idea of having the lang as GET parameter, rather than having another cookie. Advantage:

- one can directly link to a specific language version of a site.

So, I would be in favour of a PR.
 
@phochste what do you think?

**Comment by nicolasfranck** [comment]
@vpeil : the parameter "lang" in my opinion is just something to trigger the language switch (rather than using a special route for this). If you don't keep "lang" in a cookie, you would have to provide this parameter to every url you're creating (bit cumbersome). We only have two cookies for now, not so much ;-)

See branch https://github.com/LibreCat/LibreCat/tree/set_lang for this. I haven't added a PR yet, but was waiting for input.

**Comment by nicolasfranck** [comment]
Mm, I just realised that I add the parameter "lang" to the url, using the javascript history api, on page load.

**Comment by vpeil** [comment]
should I revert  my commit 5d7f029?

**Comment by nicolasfranck** [comment]
Mm, yes, because it is not clear yet how to fix this problem in a feasible way in this branch

**Comment by vpeil** [comment]
ok, reverted.

**Comment by verheyenkoen** [comment]
See #524

**Comment by vpeil** [comment]
@phochste is this solved with the merge of #524
